lib/widgets/editor/tree_widgets/commands/wait_command_widget.dart

```diff
@@ -20,37 +20,63 @@ class WaitCommandWidget extends StatelessWidget {
     this.onDuplicateCommand,
   });
 
+  void _increaseWaitTime() {
+    _updateWaitTime(command.waitTime + 0.2);
+  }
+
+  void _decreaseWaitTime() {
+    _updateWaitTime(command.waitTime - 0.2);
```

Comment:
The `NumberTextField` widget that the wait command uses already has an increment feature using the arrow keys. Its a bit less visible than this but it seems a little redundant to have it increment in 2 different places. Also, an increment of 0.1s is probably a lot better for fine control and quick adjustments.

Reply:
Yeah i guess its quite a bit redundant